docs/user_guide.md

```diff
@@ -295,6 +299,45 @@ IP2 assetName2
 
 <img alt="system_setting_asset_Ip_mapping_batch" src="images/user guide/system_setting_asset_Ip_mapping_batch2.png">
 
+#### Asset Management
+
+Here the user can manage the assets manually created by the user, but can only manage the assets created by the user through the UI.
```

Comment:
User can manage the assets created through the UI here. Other assets feed by DCIM adapters are not supported.